qubekit/fragmentation/WBOFragmenter.py

```diff
@@ -0,0 +1,72 @@
+from typing_extensions import Literal
+from typing import List, Optional, Union
+
+from pydantic import Field
+from qcelemental.util import which_import
+
+from qubekit.molecules import Ligand
+from qubekit.utils.datastructures import StageBase
+
+
+class WBOFragmenter(StageBase):
+
+    type: Literal["WBOFragmenter"] = "WBOFragmenter"
+
+    rotatable_smirks: List[str] = Field(
+        ["[!#1]~[!$(*#*)&!D1:1]-,=;!@[!$(*#*)&!D1:2]~[!#1]"],
+        description="The rotatable smirks to use for fragmentation",
+    )
+
+    @classmethod
+    def is_available(cls) -> bool:
+        fragmenter = which_import(
+            "openff.fragmenter",
+            return_bool=True,
+            raise_error=True,
+            raise_msg="Please install via `conda install -c conda-forge openff-fragmenter`.",
+        )
+
+        return fragmenter
+
+    def run(self, molecule: Ligand, **kwargs) -> Ligand:
+        from openff.toolkit.topology import Molecule as OFFMolecule
+
+        # from openff.bespokefit.executor.services.fragmenter.worker import _deduplicate_fragments
+        from openff.fragmenter.depiction import depict_fragmentation_result
+        from openff.fragmenter.fragment import WBOFragmenter
+
+        fragmenter = WBOFragmenter()
+
+        # convert to an OpenFF Molecule
+        mapped_smiles = molecule.to_smiles(mapped=True)
+        off_molecule: OFFMolecule = OFFMolecule.from_mapped_smiles(mapped_smiles)
+
+        # fragment
+        result = fragmenter.fragment(
+            off_molecule, target_bond_smarts=self.rotatable_smirks
+        )
+
+        # deduplicated_result = _deduplicate_fragments(result).json()
+
+        # simpler representation of the fragments
+        molecule.fragments = {
```

Comment:
We will need these fragments to be ligand objects to store their bond/ angle and non-bonded parameters and write them out. Maybe we could add another field to the ligand that holds the fragmenter result object with all of this info? Then the fragments field can just be a list of Ligand objects? We would then get the fragmenter info with something like 
```python
for fragment in molecule.fragments:
    torsion = fragment.fragment_result.bond_indices
```

Reply:
I saved the fragments as ligands and added "bond_indices" which might not be the best name for the field. Maybe "parent_bonds"?